docs/how-to/change-log-level.rst

```diff
@@ -0,0 +1,34 @@
+===============================
+Changing log level in microceph
```

Comment:
Nit: s/microceph/MicroCeph/

Reply:
suggestion: s/Changing/change so that it fits better in the `How to "Change log level in MicroCeph"` format.

Reply:
done.